# 365 Probabilidades — Dia #014
## Qual a probabilidade de a distância não destruir o vínculo materno?

**Tipo:** Comportamento  
**Data de publicação:** 2026-06-27  
**Ferramenta:** Python  
**Decisão analisada:** O que realmente importa para manter um vínculo forte com quem mora longe?  
**Hashtag:** #365Probabilidades #Dia014

---

### 📖 A História

Existe um medo silencioso em quem ama alguém que vai morar longe: será que a distância em quilômetros vira distância no vínculo? Será que menos tempo junto significa, automaticamente, uma conexão mais fraca?

A intuição diz que sim — mais visitas, mais ligações, mais presença física deveriam significar mais proximidade. Mas décadas de pesquisa, em dezenas de estudos diferentes, mostram algo que contraria essa intuição de um jeito reconfortante.

A frequência do contato, isoladamente, não é o que prediz o bem-estar da criança. O que prediz é outra coisa — mais sutil, mas inteiramente possível de existir mesmo a quilômetros de distância.

---

### 📚 O Conceito

A pesquisa sobre pais que não vivem com seus filhos (por divórcio, separação ou mudança) é vasta, e por décadas operou sob uma suposição simples: quanto mais contato, melhor o desenvolvimento da criança. Essa suposição foi testada repetidamente — e repetidamente refutada como explicação isolada.

A meta-análise mais influente da área, conduzida por Amato e Gilbreth (1999) e publicada no *Journal of Marriage and Family*, consolidou achados de 63 estudos diferentes. A conclusão central: **a frequência de contato com o pai não residente não estava relacionada ao bem-estar geral da criança**. O que estava associado a melhores resultados — maior sucesso acadêmico, menos problemas de comportamento — eram duas outras dimensões: **sentimento de proximidade emocional** e **parentalidade autoritativa** (presença afetuosa combinada com limites claros).

Uma atualização mais recente dessa meta-análise, com 52 estudos adicionais (Adamsons & Johnson, 2013), confirmou o mesmo padrão: a quantidade de contato e o suporte financeiro, isoladamente, não predizem bem-estar — mas o envolvimento qualitativo sim.

---

### 🧮 O Modelo
**Fontes:**
- Amato, P.R. & Gilbreth, J.G. (1999) — *Nonresident fathers and children's well-being: A meta-analysis* — Journal of Marriage and Family, 61(3) — 63 estudos consolidados
- Adamsons, K. & Johnson, S.K. (2013) — *An updated and expanded meta-analysis of nonresident fathering and child well-being* — Journal of Family Psychology, 27(4) — 52 estudos consolidados


In [ ]:
import numpy as np
import matplotlib.pyplot as plt
from scipy import stats

plt.rcParams['font.family'] = 'serif'
plt.rcParams['axes.spines.top'] = False
plt.rcParams['axes.spines.right'] = False
plt.rcParams['axes.grid'] = True
plt.rcParams['grid.alpha'] = 0.3
plt.rcParams['figure.facecolor'] = 'white'
plt.rcParams['axes.facecolor'] = 'white'

print('✅ Bibliotecas carregadas')

✅ Bibliotecas carregadas


In [ ]:
# --- DADOS DA LITERATURA ---

# Amato & Gilbreth (1999) — Journal of Marriage and Family
# Meta-analise consolidando 63 estudos sobre pais nao residentes
n_estudos_amato = 63

# Adamsons & Johnson (2013) — Journal of Family Psychology
# Replicacao e expansao com 52 estudos adicionais
n_estudos_adamsons = 52

# Dimensoes da relacao pai-filho nao residente e sua associacao com bem-estar infantil
# Classificacao qualitativa consolidada pelas duas meta-analises:
# 0 = nao associado / 1 = associado positivamente

dimensoes = {
    'Frequência de contato (visitas, ligações)': 0,
    'Suporte financeiro (pensão)': 1,
    'Proximidade emocional percebida': 1,
    'Parentalidade autoritativa (afeto + limites)': 1,
}

print('=' * 65)
print('  DADOS DA LITERATURA — VÍNCULO COM PAI/MÃE NÃO RESIDENTE')
print('=' * 65)
print(f'\n  Amato & Gilbreth 1999 — consolidação de {n_estudos_amato} estudos')
print(f'  Adamsons & Johnson 2013 — replicação com {n_estudos_adamsons} estudos adicionais')
print(f'\n  Dimensão                                    | Associada ao bem-estar?')
print(f'  {"-"*65}')
for dim, assoc in dimensoes.items():
    status = 'SIM' if assoc == 1 else 'NÃO (isoladamente)'
    print(f'  {dim:<45} | {status}')
print('=' * 65)

  DADOS DA LITERATURA — VÍNCULO COM PAI/MÃE NÃO RESIDENTE

  Amato & Gilbreth 1999 — consolidação de 63 estudos
  Adamsons & Johnson 2013 — replicação com 52 estudos adicionais

  Dimensão                                    | Associada ao bem-estar?
  -----------------------------------------------------------------
  Frequência de contato (visitas, ligações)     | NÃO (isoladamente)
  Suporte financeiro (pensão)                   | SIM
  Proximidade emocional percebida               | SIM
  Parentalidade autoritativa (afeto + limites)  | SIM


In [ ]:
# --- O MODELO ---
# Nota metodológica importante: nenhuma das duas meta-análises (Amato & Gilbreth 1999;
# Adamsons & Johnson 2013) reporta o N agregado de participantes nos resumos
# disponíveis — apenas o número de estudos consolidados (63 e 52, respectivamente).
# Por isso, o modelo usa o NÚMERO DE ESTUDOS como medida de robustez e
# REPLICAÇÃO do achado, em vez de simular um IC artificial sobre um N inventado.

total_estudos_consolidados = n_estudos_amato + n_estudos_adamsons
proporcao_dimensoes_associadas = sum(dimensoes.values()) / len(dimensoes)

print('=' * 65)
print('  MODELO — RESULTADO')
print('=' * 65)
print(f'  Total de estudos consolidados nas duas meta-análises: {total_estudos_consolidados}')
print(f'  → Replicação independente: 1999 e 2013, com 14 anos de intervalo')
print(f'  → Mesma direção do achado nas duas ocasiões')
print()
print(f'  Proporção de dimensões qualitativas associadas ao bem-estar: '
      f'{proporcao_dimensoes_associadas*100:.0f}%')
print(f'  → De 4 dimensões avaliadas, 3 mostraram associação positiva')
print(f'  → A única que NÃO mostrou associação isolada foi frequência de contato')
print()
print('  CONCLUSÃO DO MODELO:')
print('  → O que prediz bem-estar infantil não é quanto contato acontece,')
print('    é a QUALIDADE da conexão quando o contato acontece.')
print('=' * 65)

  MODELO — RESULTADO
  Total de estudos consolidados nas duas meta-análises: 115
  → Replicação independente: 1999 e 2013, com 14 anos de intervalo
  → Mesma direção do achado nas duas ocasiões

  Proporção de dimensões qualitativas associadas ao bem-estar: 75%
  → De 4 dimensões avaliadas, 3 mostraram associação positiva
  → A única que NÃO mostrou associação isolada foi frequência de contato

  CONCLUSÃO DO MODELO:
  → O que prediz bem-estar infantil não é quanto contato acontece,
    é a QUALIDADE da conexão quando o contato acontece.


In [ ]:
# --- VISUALIZACAO ---

# GRAFICO 1 — Dimensões associadas vs não associadas ao bem-estar
fig1, ax1 = plt.subplots(figsize=(12, 8))

dims_labels = list(dimensoes.keys())
dims_valores = [1 if v == 1 else 0.15 for v in dimensoes.values()]  # 0.15 para visualizar a barra "nao associada"
cores1 = ['#c0392b' if v == 0 else '#2a8a82' for v in dimensoes.values()]

bars = ax1.barh(dims_labels, dims_valores, color=cores1, alpha=0.85, height=0.5)
ax1.set_xlim(0, 1.3)
ax1.set_xticks([])
ax1.set_title('O que realmente prediz o bem-estar da criança\nAmato & Gilbreth (1999, 63 estudos) · Adamsons & Johnson (2013, 52 estudos)',
              fontsize=13, pad=15)

labels_status = ['Não associada\n(isoladamente)', 'Associada\npositivamente',
                  'Associada\npositivamente', 'Associada\npositivamente']
for bar, label in zip(bars, labels_status):
    x_pos = bar.get_width() + 0.05
    ax1.text(x_pos, bar.get_y() + bar.get_height() / 2, label,
             va='center', fontsize=11, fontweight='bold',
             color='#c0392b' if 'Não' in label else '#2a8a82')

plt.tight_layout()
plt.savefig('dia-014-grafico-01-dimensoes.png', dpi=150, bbox_inches='tight')
plt.close()
print('✅ Grafico 1 salvo!')


# GRAFICO 2 — Replicação ao longo do tempo
fig2, ax2 = plt.subplots(figsize=(12, 8))

anos = ['1999\n(Amato & Gilbreth)', '2013\n(Adamsons & Johnson)']
n_estudos_plot = [n_estudos_amato, n_estudos_adamsons]
cores2 = ['#c8a84b', '#2a8a82']

bars2 = ax2.bar(anos, n_estudos_plot, color=cores2, alpha=0.85, width=0.45)
ax2.set_ylabel('Número de estudos consolidados na meta-análise', fontsize=12)
ax2.set_title('O mesmo achado, replicado 14 anos depois\nFrequência de contato não prediz bem-estar — em ambas as meta-análises',
              fontsize=13, pad=15)
ax2.set_ylim(0, 75)

for bar, n in zip(bars2, n_estudos_plot):
    ax2.text(bar.get_x() + bar.get_width() / 2, n + 2,
             f'{n} estudos', ha='center', fontweight='bold', fontsize=14)

plt.tight_layout()
plt.savefig('dia-014-grafico-02-replicacao.png', dpi=150, bbox_inches='tight')
plt.close()
print('✅ Grafico 2 salvo!')


# GRAFICO 3 — Síntese visual do modelo
fig3, ax3 = plt.subplots(figsize=(12, 8))

categorias3 = ['Quantidade\nde contato', 'Qualidade\nda conexão']
importancia = [0.25, 0.90]  # ilustrativo, baseado na direção do achado consolidado
cores3 = ['#888888', '#2a8a82']

bars3 = ax3.bar(categorias3, importancia, color=cores3, alpha=0.85, width=0.4)
ax3.set_ylim(0, 1.1)
ax3.set_yticks([])
ax3.set_title('O que pesa mais para o bem-estar da criança\nSíntese de Amato & Gilbreth (1999) e Adamsons & Johnson (2013)',
              fontsize=13, pad=15)

labels3 = ['Não associada\nisoladamente', 'Principal\npreditor qualitativo']
for bar, label in zip(bars3, labels3):
    ax3.text(bar.get_x() + bar.get_width() / 2, bar.get_height() + 0.03,
             label, ha='center', fontweight='bold', fontsize=12)

plt.tight_layout()
plt.savefig('dia-014-grafico-03-sintese.png', dpi=150, bbox_inches='tight')
plt.close()
print('✅ Grafico 3 salvo!')

✅ Grafico 1 salvo!
✅ Grafico 2 salvo!
✅ Grafico 3 salvo!


### 💡 O Insight

Em duas meta-análises independentes, com 14 anos de intervalo entre elas e somando 115 estudos consolidados, o mesmo achado se repete: **a frequência de contato com o pai ou mãe que não mora com a criança, isoladamente, não prediz o bem-estar dela**.

O que prediz — com associação positiva consistente nas duas meta-análises — é a **proximidade emocional percebida** e a **parentalidade autoritativa**: presença afetuosa combinada com limites claros, mesmo quando o tempo junto é limitado.

Isso não significa que visitas e ligações não importam. Significa que elas são o veículo, não o destino. O que realmente constrói o vínculo é o que acontece dentro do contato — não quantas vezes ele acontece.

*Se o que mais importa é a qualidade da conexão e não a quantidade de tempo junto, o que você pode construir, mesmo a distância, que realmente fortalece esse vínculo?*

---

### ⚠️ Limitações do Modelo
- Nenhuma das duas meta-análises reporta o número agregado de participantes nos resumos disponíveis — o modelo usa o número de estudos consolidados (63 e 52) como medida de robustez, não um intervalo de confiança calculado sobre N de pessoas
- A maior parte da pesquisa nessa área foi conduzida com pais não residentes dentro do mesmo país — a generalização para situações de mudança internacional (diferença de fuso horário, idioma, cultura) requer cautela
- "Bem-estar da criança" é medido de formas diferentes entre os estudos consolidados (sucesso acadêmico, comportamento, ajuste psicológico) — a heterogeneidade de medidas é uma limitação reconhecida da própria área
- Causalidade não é estabelecida de forma definitiva: pais mais próximos emocionalmente podem ser pais que já tinham vínculos mais fortes antes da separação, não necessariamente porque agiram de forma diferente depois
- O fator de correção padrão do projeto (×0,80) não foi aplicado nesta análise, já que o modelo não trabalha com proporções diretas, e sim com a robustez da replicação entre estudos

*A ciência é honesta sobre o que não sabe. O modelo também.*

---

### 📎 Links
- Substack: [link do post]
- Instagram: [link do post]

---
*365 Probabilidades · Decidindo com dados, um dia de cada vez.*